src/sites/dMCDA.jl

```diff
@@ -286,7 +345,7 @@ end
 
 
 """
-    dMCDA(d_vars::DMCDA_vars, alg_ind::Int64, log_seed::Bool, log_shade::Bool, prefseedsites::AbstractArray{Int}, prefshadesites::AbstractArray{Int}, rankingsin::Matrix{Int64})
+    guided_site_selection(d_vars::DMCDA_vars, alg_ind::Int64, log_seed::Bool, log_shade::Bool, prefseedsites::AbstractArray{Int}, prefshadesites::AbstractArray{Int}, rankingsin::Matrix{Int64})
 
 # Arguments
 - d_vars : DMCDA_vars type struct containing weightings and criteria values for site selection.
```

Comment:
Wrap listed arguments with backticks. This is necessary for these to be correctly rendered on the documentation website

e.g.,

- \`d_vars\`
- \`alg_ind\`